README.md

```diff
@@ -97,15 +97,15 @@ By default the build script `build_helper.sh` enables the building of test targe
 ## Run a sample client and server
 Building the test targets of `mvfst` (or via `build_helper.sh`) should automatically build the sample client and server binaries as well. The server will automatically bind to `::1` by default if no host is used, but you can then spin a simple echo server by running:
 ```
-./quic/samples/echo -mode=server -host=<host> -port=<port>
+./_build/build/quic/samples/echo -mode=server -host=<host> -port=<port>
```

Comment:
`_build/build` is simply a build directory that the helper script we provide happens to install these target (and can very if users supply a different path while building). So, instead of hardcoding that path, we can perhaps clarify the textin line #98 to state that the binary targets are located in `_build/build` if built using the script `build_helper.sh` that we provide.

Reply:
@udippant 
That makes sense. I've fixed it in the current version.
If you feel like some minor changes need to be made, I've allowed edits from maintainers.